plugin.rb

```diff
@@ -108,12 +108,21 @@ def ensure_models
   end
 
   post '/IpamDriver.ReleaseAddress' do
-    Addresses::Release.run!(
-      pool_id: params['PoolID'],
-      address: params['Address']
-    )
-
-    json({})
+    begin
+      Addresses::Release.run!(
+        pool_id: params['PoolID'],
+        address: params['Address']
+      )
+
+      json({})
+    rescue Mutations::ValidationException => exc
+      if exc.errors['address'] && exc.errors['address'].symbolic == :zombie
+        status 202
```

Comment:
I don't think this should be considered as a "normal" case. It should really be logged and noticed eventually, because it's useful as a sign of some new netns leak cropping up that really also needs to get fixed. We're hopefully mitigating the damage, but you can't ignore the problem completely.

I wouldn't make this a 2xx response, because the end result of this error case is exactly the same as if the `/IpamDriver.ReleaseAddress` request had not been done at all. The IPAM rejects the request and takes no action.

Agreed that it shouldn't necessarily be the generic "HTTP 400" that things like passing an invalid IP address would return. I would perhaps consider something like a HTTP 409 Conflict?

Re HTTP 202 specifically, that seems to be for something like the `kontena service deploy` case of asynchronous request processing:

> The representation sent with this response ought to describe the request's current status and point to (or embed) a status monitor that can provide the user with an estimate of when the request will be fulfilled.